packages/react-accordion/src/components/AccordionHeader/useAccordionHeader.tsx

```diff
@@ -67,6 +72,7 @@ export const useAccordionHeader = (props: AccordionHeaderProps, ref: React.Ref<H
     expandIcon: resolveShorthand(expandIcon, {
       required: true,
       defaultProps: {
+        children: <ChevronRightRegular transform={`rotate(${mapStateToRotation({ open, expandIconPosition })})`} />,
```

Comment:
I don't see any handling for RTL, is it safe?

---

This might be bad idea, but should it be applied in styles itself? Let's say that I would like to replace this icon with `ChevronRightFilled` (just to make it filled), but this logic will not be applied to the icon and it will not rotate.

Reply:
Yeah, it appears to have been broken for RTL before this change. I updated the chevron icon rotation to account for RTL as a bonus fix in this PR.

The rotation was only applied to the default icon before my PR, and I'll leave it as-is for now. I can see how it could be helpful to have the rotation automatically applied to the slot itself, via css; but that would make it more difficult to use different custom icons based on the state. And I don't want to change Accordion's behavior in this PR.